vnext/Microsoft.ReactNative/Base/CoreNativeModules.cpp

```diff
@@ -35,6 +39,11 @@ bool HasPackageIdentity() noexcept {
   return hasPackageIdentity;
```

Comment:
just noticing now that this functionality is duplicated in 3 other places:

- https://github.com/microsoft/react-native-windows/blob/0266f65ffeb0eef353612f9fedb89e3d5bfc6abb/vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp#L473
- https://github.com/microsoft/react-native-windows/blob/main/vnext/Shared/DevSupportManager.cpp#L252
- https://github.com/microsoft/react-native-windows/blob/main/vnext/Shared/Utils.cpp#L60

Consider consolidating into a shared location